src/hooks/screen/useSignedChatInfoHook.ts

```diff
@@ -52,4 +53,4 @@ function useAnonymousChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
   };
 }
 
-export default useAnonymousChatInfoScreenHook;
+export default useSignedChatInfoScreenHook;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The exported function name has been updated from `useAnonymousChatInfoScreenHook` to `useSignedChatInfoScreenHook`. This is consistent with the function name change.


<!-- This is an auto-generated comment by OSS CodeRabbit -->